packages/bits/demo/src/components/demo/checkbox-group/checkbox-group-in-form/checkbox-group-in-form.example.component.ts

```diff
@@ -18,30 +18,28 @@
 //  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 //  THE SOFTWARE.
 
-import { Component, OnInit } from "@angular/core";
-import { FormBuilder, FormGroup, Validators } from "@angular/forms";
+import { Component } from "@angular/core";
+import { FormBuilder, Validators } from "@angular/forms";
 
 import { ToastService } from "@nova-ui/bits";
 
 @Component({
     selector: "nui-checkbox-group-in-form-example",
     templateUrl: "./checkbox-group-in-form.example.component.html",
 })
-export class CheckboxGroupInFormExampleComponent implements OnInit {
-    public myForm: FormGroup;
+export class CheckboxGroupInFormExampleComponent {
     public cabbage = $localize`Cabbage`;
     public potato = $localize`Potato`;
     public tomato = $localize`Tomato`;
     public carrot = $localize`Carrot`;
     public vegetables = [this.cabbage, this.potato, this.tomato, this.carrot];
     public selectedVegetables = [this.cabbage];
+    public myForm;
 
     constructor(
         private formBuilder: FormBuilder,
         private toastService: ToastService
-    ) {}
-
-    public ngOnInit(): void {
```

Comment:
That was moved because of the form initialization. ES2020 has changed it and we can't initialize form in fields section. Neither in ngOninit because then typescript will not recognize the typed form. Same works for rest of the places.